components/molecule/avatar/src/AvatarFallbackIcon/index.scss

```diff
@@ -0,0 +1,7 @@
+$c-avatar-fallback-icon: inherit !default;
+
+.react-MoleculeAvatarFallbackIcon {
```

Comment:
Namespace should be `sui` instead `react`.

Reply:
Change made! thank you @midudev